modernizer-maven-plugin/src/main/resources/modernizer.xml

```diff
@@ -1250,7 +1250,7 @@ violation names use the same format that javap emits.
   <violation>
     <name>java/io/FileOutputStream."&lt;init&gt;":(Ljava/lang/String;Z)V</name>
     <version>7</version>
-    <comment>Prefer java.nio.file.Files.newOutputStream(java.nio.file.Paths.get(String), java.nio.file.StandardOpenOption.CREATE, java.nio.file.StandardOpenOption.CREATE)</comment>
+    <comment>Prefer java.nio.file.Files.newOutputStream(java.nio.file.Paths.get(String), CREATE, APPEND, WRITE)</comment>
```

Comment:
Are you sure that `new FileOutputStream(name)` (without `append` parameter) actually is *appending*? 🤔 The JavaDocs do neither confirm nor deny this.